compiler/fixtures/runtime-compat/app-18/package.json

```diff
@@ -11,13 +11,16 @@
   },
   "dependencies": {
     "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "react-compiler-runtime": "0.0.0-experimental-8d8e73f-20241009",
```

Comment:
This is only required for the app code, not to make the library work, right? Since the library has its own dependency on the compiler runtime.

Reply:
yup exactly